cmake/SetupCodeChecks.cmake

```diff
@@ -417,8 +417,8 @@ macro(blt_add_uncrustify_target)
             COMMAND ${UNCRUSTIFY_EXECUTABLE} --version
             OUTPUT_VARIABLE _version_str
             OUTPUT_STRIP_TRAILING_WHITESPACE )
-        string(REGEX MATCH "([0-9]+(\\.)?)+(_[a-zA-Z])?$" _uncrustify_version ${_version_str})
-        
+        string(REGEX MATCH "([0-9]+(\\.)?)+(_[a-zA-Z])?" _uncrustify_version ${_version_str})
```

Comment:
@rrsettgast -- You updated this regex last.  It looks like this change should work for you.